packages/fxa-settings/src/pages/Pair/Index/index.test.tsx

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { fireEvent, screen } from '@testing-library/react';
+import { renderWithRouter } from '../../../models/mocks';
+// import { getFtlBundle, testAllL10n } from 'fxa-react/lib/test-utils';
+// import { FluentBundle } from '@fluent/bundle';
+import { usePageViewEvent } from '../../../lib/metrics';
+import { REACT_ENTRYPOINT, ENTRYPOINTS } from '../../../constants';
+import { MOCK_ERROR } from './mocks';
+import Pair, { viewName } from '.';
+
+jest.mock('../../../lib/metrics', () => ({
+  usePageViewEvent: jest.fn(),
+}));
+
+describe('Pair', () => {
+  // let bundle: FluentBundle;
+  let mockCallback: Function;
+  beforeAll(async () => {
+    //   bundle = await getFtlBundle('settings');
+    mockCallback = jest.fn();
```

Comment:
Does the mockCallback need to be async? And do we need to clear mocks in any way?

Reply:
Oh I see what's going on here -- this is `async` anticipating using `bundle = await` later. I'll fix it for now. I don't think we need to clear these mocks since they're in the `beforeAll`?

Reply:
You're most likely right, I'm still learning how to test with Jest.

Reply:
I could be wrong ....I'm going to look into this one

Reply:
Hmmmm it looks we could set something in the config, but there _IS_ a precedent I had never noticed before of clearing them in some but not all Settings pages -- I'll add it in! Thank you for asking this!